integration/tests/epochs/epoch_test.go

```diff
@@ -131,79 +126,49 @@ func (s *Suite) TestViewsProgress() {
 	}
 }
 
-// TestEpochJoin will asserts that a node can successfully join the protocol by staking during
-// the Epoch Staking Phase and asserting that node info read from the staking table match the
-// node info generated by the test
-func (s *Suite) TestEpochJoin() {
+// TestDynamicProtocolParticipants will stake a node by submitting all the transactions
+// that an node operator would submit, start a new container for that node, and remove
+// a container from the network of the same node type. After this orchestration assertions
+// specific to that node type are made to ensure the network is healthy.
+func (s *Suite) TestDynamicProtocolParticipants() {
```

Comment:
nit: not crazy about the name - I find it kind of confusing. Can it be something like `TestEpochJoinAndLeave`

Reply:
https://github.com/onflow/flow-go/pull/1728/commits/f3a7b7c6f13162e77aa439a1e15514dacbe899fe